runtime/ejs-webgl.m

```diff
@@ -1061,6 +1061,53 @@ void copyTexSubImage2D(GLenum target, GLint level, GLint xoffset, GLint yoffset,
     return NUMBER_TO_EJSVAL(rv);
 }
 
+JSMETHOD(getParameter) {
+	if (argc != 1)
+		THROW_ARG_COUNT_EXCEPTION(1);
+
+        GLenum pname = (GLenum)EJSVAL_TO_NUMBER(args[0]);
+        switch (pname) {
+            case GL_MAX_TEXTURE_IMAGE_UNITS:
+            case GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS:
+            case GL_MAX_VERTEX_ATTRIBS:
+            case GL_MAX_VERTEX_UNIFORM_VECTORS:
+            case GL_MAX_VARYING_VECTORS:
+            case GL_MAX_FRAGMENT_UNIFORM_VECTORS:
+            case GL_MAX_CUBE_MAP_TEXTURE_SIZE:
+            case GL_MAX_TEXTURE_SIZE: {
+                GLint intData = 0;
+                glGetIntegerv (pname, &intData);
+                return NUMBER_TO_EJSVAL(intData);
+            }
+            case GL_IMPLEMENTATION_COLOR_READ_FORMAT:
+                /* GL_RGBA */
+                /* calberto's note: glGetIntegerv was failing here. Fix later. */
```

Comment:
let's use `XXX(calberto)` here - XXX is easier to search for, and finding your own is as easy as searching for `XXX(calberto)` :)